instagram/src/App.js

```diff
@@ -9,19 +9,34 @@ class App extends Component {
     super();
     this.state ={
       data: dummyData,
+      filteredUsers: [],
+      searchInput: '',
     }
   }
+
```

Comment:
You should be using componentDidMount() {
}

To populate your data property on state.